torch/csrc/jit/ir/irparser.cpp

```diff
@@ -226,12 +229,41 @@ ParsedLiteral IRParser::parseScalarLiteral(Node* n) {
       r.k = AttributeKind::t;
       return r;
     }
+    case '{': {
+      L.next();
+      if (L.cur().kind == '-') {
+        L.next();
+      }
+      auto text = L.expect(TK_NUMBER);
```

Comment:
it would be easy enough to actually recover the exact value here... but I think more consistent to not, as you are currently doing